lib/emitter/src/Events/AssignmentGraded.php

```diff
@@ -58,14 +58,7 @@ public function read(array $opts) {
         ]);
 
         // Excluded from array merge to make sure that the actor is overwritten e.g. if a different IFI is used.
-        $statement['actor'] = [
-            'objectType' => 'Agent',
-            'name' => $opts['graded_user_name'],
-            'account' => [
-                'homePage' => $opts['graded_user_url'],
-                'name' => $opts['graded_user_id'],
-            ],
-        ];
+        $statement['actor'] = $this->readUser($opts, "graded_user");
```

Comment:
I think the tests are failing because `readUser` should be `read_user`, try changing that @Nicosoft and see if that works.